### PR TITLE
updated client libs

### PR DESCRIPTION
**Pull request by dcchivian**

### CODE REVIEW

**Comment by dcchivian** [comment]
passing all unit tests

File: lib/kb_ReadsUtilities/kb_ReadsUtilitiesImpl.py [hunk 1]
```diff
@@ -66,9 +65,22 @@ class kb_ReadsUtilities:
...
 
 
-    # target is a list for collecting log messages                                                                                
+    def now_ISO(self):
+        now_timestamp = datetime.now()
+        now_secs_from_epoch = (now_timestamp - datetime(1970,1,1)).total_seconds()
+        now_timestamp_in_iso = datetime.fromtimestamp(int(now_secs_from_epoch)).strftime('%Y-%m-%d_%T')
+        return now_timestamp_in_iso
+
+    def log(self, target, message):
```
Comment (bio-boris) [inline lib/kb_ReadsUtilities/kb_ReadsUtilitiesImpl.py]: Log function is defined twice

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+### Version 1.1.0
+__Changes__
+- patched "bad status line" error with newer kb-sdk install of SDK_LOCAL clients
+- added generate_microbiome_in_silico_reads_from_real_reads() method
```
Comment (bio-boris) [inline RELEASE_NOTES.md]: I can't seem to find this function. Did you include this or did I just not find it?

Comment (dcchivian) [reply RELEASE_NOTES.md]: I got ahead of myself.  will add function next, but not as part of this PR

File: lib/kb_ReadsUtilities/kb_ReadsUtilitiesImpl.py (hunk 1 above)
Comment (dcchivian) [reply lib/kb_ReadsUtilities/kb_ReadsUtilitiesImpl.py]: not anymore!  silly mistake by me

**Pull request merged**